orderlyweb_api/orderly_web_api.py

```diff
@@ -9,21 +11,40 @@ def __init__(self, base_url, token):
         response = requests.post(auth_url, headers=headers)
         if response.status_code != 200:
             msg = 'Unexpected status code: {}. Unable to authenticate.'
-            raise Exception(msg.format(response.status_code))
+            raise OrderlyWebResponseError(msg.format(response.status_code),
+                                          response=response)
         self.token = response.json()['access_token']
```

Comment:
So `self.token` is an OW access token, and the `token` that is passed to `__init__` is a Montagu token? Perhaps relname one or both to make this super clear.